src/TagsInput/components/TagsContainer.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import * as React from 'react';
+import { View, ScrollView } from 'react-native';
+import { Badge } from '../../Badge';
+import { StyleSheet } from '../../PlatformStyleSheet';
+import { MIN_WIDTH } from '../constants';
+
+type Props = {|
+  +tags?: string[],
+  +fontSize: number,
+  +maxWidth: string | number,
+|};
+
+class TagsContainer extends React.PureComponent<Props> {
+  static defaultProps = {
+    maxWidth: 'auto',
+  };
+
+  render() {
+    const { tags, fontSize, maxWidth } = this.props;
+    if (!tags || tags.length < 1) {
+      return null;
+    }
+
+    return (
+      <View>
```

Comment:
Is this wrapping <View> necessary?

Reply:
It is, I need it for the web, if I don't use it break how overflow works